### PR TITLE
Romelchenko oop - Push homework5 files to master branch

### PR DESCRIPTION
**Pull request by RomanO-qa**

### CODE REVIEW

File: Bank/bank.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        self.name = name
+        self.list_of_accounts = []
+
+    def add_account(self, account):
+        self.list_of_accounts.append(account)
+
+    def remove_account(self, account):
+        self.list_of_accounts.remove(account)
+
+    def all_bank_accounts(self, account):
```
Comment (toleksyn) [inline Bank/bank.py:16]: rename

File: Bank/bank.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        print(self.name, "has such account holders")
+        for account in self.list_of_accounts:
+            print(account.account_holder_name, "with", account.balance, "balance")
+
+    def find_account(self, account_number):
```
Comment (toleksyn) [inline Bank/bank.py:21]: rename

File: Bank/bank.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        for account in self.list_of_accounts:
+            if account.account_number == account_number:
+                return "Account holder: " + account.account_holder_name
+        return "Account with " + "'" + str(account.account_number) + "'" + " account number doesn't exist"
+
+    def total_interests_earned(self):
```
Comment (toleksyn) [inline Bank/bank.py:27]: rename

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: Bank/bank.py (hunk 1 above)
Comment (RomanO-qa) [reply Bank/bank.py]: renamed to bank_info

File: Bank/bank.py (hunk 3 above)
Comment (RomanO-qa) [reply Bank/bank.py]: renamed to earned_interests

File: Bank/account.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+class Account:
+
+    def __init__(self, account_number, account_holder_name, balance, interest_rate):
+        self.account_number = account_number
+        self.account_holder_name = account_holder_name
+        self.balance = balance
+        self.interest_rate = interest_rate
+
+    def deposit_money(self):
+        print("Deposit money!")
```
Comment (toleksyn) [inline Bank/account.py:15]: should probably use a member variable and store deposit amount there

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: Bank/account.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+class Account:
+
+    def __init__(self, account_number, account_holder_name, balance, interest_rate):
+        self.account_number = account_number
+        self.account_holder_name = account_holder_name
+        self.balance = balance
+        self.interest_rate = interest_rate
+
+    def deposit_money(self):
```
Comment (toleksyn) [inline Bank/account.py:14]: deposit_money(sum)

**Pull request closed without merging**